### PR TITLE
gather: copy event-filter.html out of must-gather archive

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Review by stevekuznetsov — APPROVED** [review]

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ deck:
         name: html
       required_files:
       - .*/e2e-intervals.*\.html
+      - .*/event-filter\.html
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_config.yaml:81]: /hold

Comment (alvaroaleman) [reply core-services/prow/02_config/_config.yaml:81]: I am very sure this is not what you want, now you require both of them. You want to OR them, right? If so, it must be one regex

Comment (mfojtik) [reply core-services/prow/02_config/_config.yaml:81]: ack, thanks @alvaroaleman

**Comment by mfojtik** [comment]
@deads2k can you lgtm again?

**Pull request merged**